src/components/label/label.wp.scss

```diff
@@ -61,8 +61,8 @@ html[dir="rtl"] .label-wp[floating] {
   transform: translate3d(0, 0, 0) scale(.8);
 }
 
-html[dir="rtl"] .input-has-focus .label-wp[floating],
-html[dir="rtl"] .input-has-value .label-wp[floating] {
+[dir="rtl"] .input-has-focus .label-wp[floating],
```

Comment:
Easy,
unfortunately the browser will choose this rule
```
[dir="rtl"] .label-wp[floating] {
  transform: translate3d(-8px, 34px, 0);
}
```
over this one
```
.input-has-focus .label-wp[floating],
.input-has-value .label-wp[floating] {
  transform: translate3d(0, 0, 0) scale(.8);
}
```
even if the label has a parent element with `input-has-focus` class because of the attribute specification from previous rule which has a higher priority than a class. so I've added exactly the same thing over again with attribute specification, so the browser choose the right one on focus or when the input has value when in rtl mode....